File: week_1/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = height; i > 0; i--)
    {

        for (int j = 0; j < i; j++)
        {
            printf(" ");
        }

        for (int a = i; a <= height; a++)
        {

            printf("#");
        }
        printf("\n");
    }
}

File: week_6/mario.py
from cs50 import get_int

while True:
    height = get_int("Height for  side: ")
    if height > 0 and height <= 8:
        break
    else:
        pass


for i in range(height, 0, -1):
    for j in range(i - 1):
        print(" ", end="")
    for k in range(0, height - i + 1):
        print("#", end="")
    for l in range(2):
        print(" ", end="")
    for m in range(0, height - i + 1):
        print("#", end="")
    print()
